<!DOCTYPE html>
<html>
	<head>
		<meta charset="UTF-8">
		<title>exception_try_catch</title>
		<script>
			try {
				willExcept.byeBye(); // willExcept 객체 및 byeBye() 메소드에서 예외 발생  
				console.log("try 구문의 마지막 줄"); // 위에서 예외가 발생하므로 실행되지 않고 catch 구문으로 이동
			} catch (exception) {
				console.log("catch 구문의 마지막 줄");
			}
		</script>
	</head>
	<body>
		
	</body>
</html>